### PR TITLE
typo correction

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user report #8285, so i corrected word

### CODE REVIEW

File: microsoft-365/admin/add-users/intune-admin-roles-in-the-mac.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ In the Microsoft 365 admin center, you can go to **Roles**, and then select any
 
 |Admin role     |Who should be assigned this role?  |
 |---------|---------|
-|Application manager     |   Assign the Application manager role to users who mangage the application lifecycle for mobile apps, configures policy-managed apps and views device info and configuration profiles.  |
+|Application manager     |   Assign the Application manager role to users who manage the application lifecycle for mobile apps, configures policy-managed apps and views device info and configuration profiles.  |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/intune-admin-roles-in-the-mac.md]:
```suggestion
|Application manager     |   Assign the Application manager role to users who manage the application lifecycle for mobile apps, configure policy-managed apps, and view device info and configuration profiles.  |
```

File: microsoft-365/admin/add-users/intune-admin-roles-in-the-mac.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ In the Microsoft 365 admin center, you can go to **Roles**, and then select any
...
 |Help desk operator     |   Assign the help desk operator role to users who assign apps and policies to users and devices. |
 |Intune role administrator    |   Assign the Intune role administrator to users who can assign Intune permissions to other admins and can manage custom and built in Intune roles.   |
 |Policy and profile manager     |   Assign the policy and profile manager role to users manage compliance policy, configuration profiles and Apple enrollment.   |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/intune-admin-roles-in-the-mac.md:40]:
```suggestion
|Intune role administrator    |   Assign the Intune administrator role to users who assign Intune permissions to other admins and manage custom and built-in Intune roles.   |
|Policy and profile manager     |   Assign the policy and profile manager role to users who manage compliance policy, configuration profiles, and Apple enrollment.   |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**